pkg/vpcmodel/commonConnectivity.go

```diff
@@ -6,7 +6,139 @@ SPDX-License-Identifier: Apache-2.0
 
 package vpcmodel
 
-import "github.com/np-guard/models/pkg/connection"
+import (
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/netp"
+)
+
+// todo: remove stateful from connection.Set (for both options)
+
+// ConnWithStateful connection details
+type ConnWithStateful struct {
```

Comment:
move this new type and its methods/functions to a separate file, such as `conn_with_stateful.go` (golang file names conventions are with underscores).

Reply:
done

Reply:
I see you renamed and moved to `detaildConn.go`  -the golang file name convention is with underscore, so better use `detailed_conn.go` (for example see https://blog.emka.web.id/2022/09/project-filename-convention-in-golang.html) 

also, the previous version was `connWithStateful` which is more specific than `detailedConn` - why renaming?

Reply:
1. I followed our named convention - e.g. `externalNetwork, debugOutput, nodesConnectivity`  and more. If I'll change it as you suggest we will have unconsistency in the package, which I find more troubling. If we are not consistent with go's naming convention, best to have a separate issue and rename all relevant files.
2. I renamed as result of Haim's CR; I agreed with him it is better not to have with as part of the name. Maybe `connStatefulDetail`? a bit too long? `connStateful` is a bit misleading I think

Reply:
opened issue #618